lib/improver/tests/utilities/test_load.py

```diff
@@ -345,13 +351,15 @@ def test_no_partial_merge_list_args(self):
         loading CLIs."""
         low_cloud_cube = self.cube.copy()
         low_cloud_cube.rename("low_type_cloud_area_fraction")
+        low_cloud_cube.units = 1
         low_cloud_cube.coord("time").points = (
             low_cloud_cube.coord("time").points + 3600)
         low_cloud_cube.coord("forecast_period").points = (
             low_cloud_cube.coord("forecast_period").points - 3600)
         save_netcdf(low_cloud_cube, self.low_cloud_filepath)
         medium_cloud_cube = self.cube.copy()
         medium_cloud_cube.rename("medium_type_cloud_area_fraction")
+        medium_cloud_cube.units = 1
```

Comment:
I feel like this test module needs a little tidying ... however perhaps not in this PR!

Reply:
Yes. The cubes are just copied from temperature and minimally modified. However, as they go through the save wrapper they fall foul of the metadata checks and require at least the slightly improved level of self-consistency.